content/src/cloud/features/cloud-backup-about.md

```diff
@@ -28,7 +28,7 @@ Storing your latest [Home Assistant backup](https://www.home-assistant.io/common
 
 ### Encrypted and private
 
-Home Assistant Cloud is the privacy-focused cloud. Nabu Casa uses Cloudflare as its storage provider for Home Assistant Cloud Backups. It specifically uses Cloudflare’s West Europe servers due to high privacy standards compared to other regions.
+Home Assistant Cloud is the privacy-focused cloud. Nabu Casa uses Cloudflare as its storage provider to store backups on Home Assistant Cloud. It specifically uses Cloudflare’s West Europe servers due to high privacy standards compared to other regions.
```

Comment:
```suggestion
Home Assistant Cloud is the privacy-focused cloud. Nabu Casa uses Cloudflare as its storage provider to upload backups to Home Assistant Cloud. It specifically uses Cloudflare’s West Europe servers due to high privacy standards compared to other regions.
```
unsure.

Reply:
maybe I misunderstood. Is Cloudflare doing the uploading? or providing the storage space?

Reply:
Im the confused one, I only read the last part.

Reply:
They only provide the storage.